composer.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "drupal/redirect",
+  "description": "Sandbox for Drupal 8 Redirect Port",
```

Comment:
This will eventually move back to d.o, just give it the right name now? The module name is Redirect here as well.